pkg/services/notifications/smtp.go

```diff
@@ -65,6 +65,10 @@ func (sc *SmtpClient) buildEmail(msg *Message) *gomail.Message {
 	m.SetHeader("From", msg.From)
 	m.SetHeader("To", msg.To...)
 	m.SetHeader("Subject", msg.Subject)
+	// add all static headers to the email message
+	for h, val := range sc.cfg.StaticHeaders {
```

Comment:
I wonder if we should reverse the order here, and apply the static headers before the From/To/Subject ones?

Only rationale being, it opens up some odd paths like overriding `From` globally to something unusual. It seems like we'd always want to respect the individual message over the static headers in this case.

Since this all comes from the config.ini anyway, and isn't UI-editable, it's probably not a big deal, so consider this a nitpick if we disagree.

Reply:
> Only rationale being, it opens up some odd paths like overriding `From` globally to something unusual.

That makes a lot of sense IMO - we should definitely disallow that.

Reply:
Yeah, may as well change the order. Doesn't hurt 👍